distributed/protocol/numpy.py

```diff
@@ -4,12 +4,6 @@
 
 import numpy as np
 
-try:
-    import blosc
-    n = blosc.set_nthreads(2)
-except ImportError:
-    blosc = False
```

Comment:
It is important to keep this code.  Blosc can improve performance when moving large memoryviews around, but we don't want it taking up all of our cores.

Reply:
> compression.py handles the blosc side of things now.

Ah, you're right.  +1 on this change.  Thanks for pushing back.

Reply:
No problem, apologize if I seemed curt.

Reply:
Curtness is welcome.  The "Thanks" comment was genuine.

Reply:
(also, you didn't seem curt)